docs/04-guides/get-viewer-engagement/_tabs.mdx

```diff
@@ -0,0 +1,33 @@
+import TabbedPages from "@site/src/components/TabbedPages";
+
+<head>
+  <title>Get viewer engagement | Livepeer Studio Docs</title>
+  <meta
+    property="og:title"
+    content="Get viewer engagement | Livepeer Studio Docs"
+  />
+</head>
+
+# Get viewer engagement
+
+This guide provides information about registering and retrieving views from a video using
+the [dashboard](./dashboard) or the [Livepeer player API](./api).
+
+## Register views
+
+To display viewership data, you need to configure your player first.
+We recommend using the [Livepeer player](https://livepeerjs.org/) as the player
+is already configured. You can follow this [guide](guides/playback-a-video-stream) to
+setup the Livepeer player.
```

Comment:
Suggestion is mssing a link to the support email like we have in other docs and formatting to break lines properly:
```suggestion
To display viewership data, you need to configure your player first.
Currently the viewership data is only registered if you're using the [Livepeer player](https://livepeerjs.org/). It also comes fully configured for the best playback experience
with Livepeer, so it's the recommended way to get started. You can follow this [guide](guides/playback-a-video-stream) to
setup the Livepeer player.

We are currently working on support for viewership data using custom players as well.
If you have any urgent help with this get in touch with our customer support
anyway and we may be able to assist you.
```